scripts/installer.nsh

```diff
@@ -0,0 +1,8 @@
+# This macro fetches the currently install MM version via MSI (if any) and uninstalls it first
+!macro customInit
+  nsExec::ExecToStack "$\"powershell.exe$\" -command $\"$$Installer = New-Object -ComObject WindowsInstaller.Installer; $$MMProduct = $$Installer.ProductsEx('', '', 7) | Where-Object -FilterScript {$$_.InstallProperty('ProductName') -eq 'Mattermost'}; if ($$MMProduct -ne $$null) {Write-Host -NoNewline $$MMProduct.ProductCode()}$\""
+  Pop $0
+  Pop $1
+  StrCmp $1 "" 0 +1
+  ExecWait '"msiexec.exe" /x $1 /qn'
```

Comment:
What's the working directory here and is it included in the search path when locating `msiexec.exe`? I.e. can we be sure we're actually executing the thing we're supposed to? The same question goes for the `powershell.exe` invocation above.

Reply:
I would guess that it would be wherever the installer is running from, which for most people would be wherever the update is downloaded to.
All of those are likely being picked up from the local PATH variable.

I'm not sure if they can be overridden, Windows might have a thing to stop that, but if this is a concern I think it makes sense to just add the absolute paths.